api_examples.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+
+class APIExampleMixin:
+    """A TestCase mixin that allows access to API by Example retrieval methods."""
+
+    SERVER_NAME = 'localhost:8000'
+
+    def create_request(self, *args, **kwargs):
+        return super().create_request(*args, SERVER_NAME=self.SERVER_NAME, **kwargs)
```

Comment:
I'm not entirely sure about keeping this method. It ties into [`incuna-test-utils`](https://github.com/incuna/incuna-test-utils/blob/master/incuna_test_utils/testcases/api_request.py#L14-L24) very well, but isn't any use without it.